src/components/AddToCalendarButton/icsGenerator.ts

```
@@ -2,22 +2,27 @@
   // Set end time to 1 hour after start time
   const endDate = new Date(startDate.getTime() + 60 * 60 * 1000);
-  
+
   // Format dates for ICS file (YYYYMMDDTHHMMSSZ format)
   const formatDateForICS = (date: Date) => {
-    return date.toISOString().replace(/-|:|\.\d+/g, '').slice(0, 15) + 'Z';
+    return (
+      date
+        .toISOString()
+        .replace(/-|:|\.\d+/g, '')
+        .slice(0, 15) + 'Z'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)